pkg/tiller/release_server.go

```diff
@@ -251,6 +251,16 @@ func (s *ReleaseServer) renderResources(ch *chart.Chart, values chartutil.Values
 		return nil, nil, "", fmt.Errorf("Chart incompatible with Tiller %s", sver)
 	}
 
+	cap, ok := values["Capabilities"].(*chartutil.Capabilities)
```

Comment:
Are we sure we want to error here if we cannot get capabilities from k8s? It seems that it should be available at all times but if there is no `k8sVersion` field to check against it seems like a wasted call (that might cause problems in edge cases).